modules/search/pages/create-search-indexes.adoc

```diff
@@ -16,20 +17,33 @@ You can copy the JSON payload from the UI to create your index with the REST API
 [#ui]
 == Creating a Search Index with the Capella UI
 
-To xref:create-search-index-ui.adoc[create a basic Search index], you only need to provide the following information: 
+To xref:create-search-index-ui.adoc[create a basic Search index], simply provide the following information:
 
-* The name of the index. 
-* The bucket where you want to create the index. 
-* The scope and collection where you want to create the index.
-* A xref:customize-index.adoc#type-mappings[type mapping] for the documents that you want to add to the index.
+* The desired index name. 
+* The desired collection (or set of collections) under a single bucket and scope in your database.
+//* The bucket, scope and collection where you want to create the index.
+//* A xref:customize-index.adoc#type-mappings[type mapping] for the documents that you want to add to the index.
 
-After you create a Search index, you can customize it to improve search results and performance.
-For more information about how you can customize a Search index with the Capella UI, see xref:customize-index.adoc[].
+Once the index is created, the system streams data from the chosen collection(s) plus all subsequent mutations into the index builder.
+
+Search indexes can be editied and further customized or modified to enhance search results and performance. For more details on customizing a Search Index using the Capella UI, refer to xref:customize-index.adoc[Customize a Search Index with the Capella UI].
+
+[#sdks]
```

Comment:
This is a much longer-term project and I would feel better having this commented out for now.